src/pages/cli/usage/export-to-cdk.mdx

```diff
@@ -1,3 +1,76 @@
-## Export an Amplify backend as a CDK project
+export const meta = {
+  title: `Export Amplify project to CDK`,
+  description: `Export Amplify CLI-generated backends as a Cloud Development Kit (CDK) stack and incorporate into existing CDK deployment pipelines. This capability allows frontend developers to build their app backend quickly and, each time it is ready to ship, hand it over to DevOps teams to deploy to production.`,
```

Comment:
"Export Amplify CLI-generated backends as a Cloud Development Kit (CDK) stack and incorporate into existing CDK deployment pipelines."

This sentence feels a little awkward to me. Here's a minor suggestion, but feel free to disregard

```suggestion
  description: `Export your Amplify CLI-generated backends as a Cloud Development Kit (CDK) stack and incorporate it into existing CDK deployment pipelines. This capability allows frontend developers to build their app backend quickly and, each time it is ready to ship, hand it over to DevOps teams to deploy to production.`,
```